app/models/city.rb

```diff
@@ -0,0 +1,3 @@
+class City < ApplicationRecord
+    has_many :places
+end
```

Comment:
Could perhaps validate the presence of the name and add a test for that?

Reply:
done